programs/mango-v4/src/instructions/account_edit.rs

```diff
@@ -18,28 +22,26 @@ pub fn account_edit(
 
     let mut account = ctx.accounts.account.load_full_mut()?;
 
-    // note: unchanged fields are inline, and match exact definition in create_account
-    // please maintain, and don't remove, makes it easy to reason about which support modification by owner
-
     if let Some(name) = name_opt {
         account.fixed.name = fill_from_str(&name)?;
     }
 
-    // unchanged -
-    // owner
-    // account_num
-    // bump
-
     if let Some(delegate) = delegate_opt {
         account.fixed.delegate = delegate;
     }
 
-    // unchanged -
-    // tokens
-    // serum3
-    // perps
-    // being_liquidated
-    // is_bankrupt
+    match (temporary_delegate_opt, temporary_delegate_expiry_opt) {
+        (Some(temporary_delegate), Some(temporary_delegate_expiry)) => {
```

Comment:
optional: This bundling of the two flags doesn't seem necessary. I'd be fine with allowing them to be set independently.